model/ftn/w3initmd.ftn

```diff
@@ -1060,6 +1078,57 @@
 !
         END DO
 !
+! FOR CHECKPOINT (J=8): Second stream of restart files
```

Comment:
same as above, do we want to label this as checkpoint?

Reply:
@JessicaMeixner-NOAA see reply above

Reply:
Changed

Reply:
@RobertoPadilla-NOAA added to thread for awareness